lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -43,16 +42,17 @@ public:
     GetId() const noexcept;
 
     /**
-     * @brief Configure the session for use.
+     * @brief Configure the session for use. Also moves the callbacks into ownership here
```

Comment:
I'm not entirely sure about the behavior of shared/weak pointers, but my understanding is that if a shared pointer is passed into this function, the ownership actually does NOT transfer.